invisible_cities/reco/paolina_functions_test.py

```diff
@@ -469,4 +475,47 @@ def test_voxel_is_dropped(hits, requested_voxel_dimensions):
     drop_voxel(voxels, voxel_to_drop)
 
     assert len(voxels) == n_starting_voxels - 1
+
+
+@given(bunch_of_hits, box_sizes, min_n_of_voxels)
+def test_total_energy_is_conserved(hits, requested_voxel_dimensions, min_voxels):
```

Comment:
This test name should contain 'drop' somewhere.

Reply:
Done.